options.go

```diff
@@ -265,3 +265,15 @@ func (o *Options) CLI() serpent.OptionSet {
 		},
 	}
 }
+
+func (o *Options) Markdown() string {
+	cliOptions := o.CLI()
+	mkd := "| Flag | Env | Default | Description |\n" +
+		"| - | - | - | - |\n"
+
+	for _, opt := range cliOptions {
+		mkd += "| " + opt.Flag + " | " + opt.Env + " | " + opt.Default + " | " + opt.Description + " |\n"
```

Comment:
```suggestion
		mkd += "| `--" + opt.Flag + "` | `" + opt.Env + "` | " + opt.Default + " | " + opt.Description + " |\n"
```
I am not sure if this is what we need :D

Reply:
Came here to say this as well 👍